modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/test/util/HeadlessBuilderTestUtil.java

```diff
@@ -45,4 +61,38 @@ public static ObjectDefinition publishObjectDefinition(
 			userId, objectDefinition.getObjectDefinitionId());
 	}
 
+	public enum ParserConstants {
```

Comment:
As in previous chapters :trollface: , I don't see the benefits of having an enumeration instead of some contants

Reply:
Well, as in previous chapters :trollface: the benefit from my point of view is to have a cleaner code because having these constants in an "enum parser" means that the constants of that enum **are the only ones** related to the parser variables used in the `rest-openapi.yaml`. 

[Here](https://www.baeldung.com/java-constants-good-practices#1-general-good-practices) and [here](https://softwareengineering.stackexchange.com/a/290009) are some resources that say what I am talking about.

Maybe my use of enums is a loosed battle.
I've changed it on the PR to use constants, I don't want more chapters in this book 😂